include/mata/nfa.hh

```diff
@@ -298,12 +288,12 @@ struct Move {
 };
 
 /// List of transitions from a certain state. Each element holds transitions with a certain symbol.
-using Moves = Mata::Util::OrdVector<Move>;
+        using Moves = Mata::Util::OrdVector<Move>;
 /// Transition relation for an NFA. Each index 'i' to the vector represents a state 'i' in the automaton.
-using TransitionRelation = std::vector<Moves>;
+        using TransitionRelation = std::vector<Moves>;
 
 /// An epsilon symbol which is now defined as the maximal value of data type used for symbols.
-constexpr Symbol EPSILON = limits.maxSymbol;
+        constexpr Symbol EPSILON = limits.maxSymbol;
```

Comment:
Were the three changes above intentional or just accidental change of indent?

Reply:
probably, no ideal how it happened